datafusion/substrait/src/logical_plan/consumer.rs

```diff
@@ -1137,7 +1220,8 @@ fn from_substrait_type(dt: &substrait::proto::Type) -> Result<DataType> {
                     from_substrait_type(list.r#type.as_ref().ok_or_else(|| {
                         substrait_datafusion_err!("List type must have inner type")
                     })?)?;
-                let field = Arc::new(Field::new("list_item", inner_type, true));
+                // "item" as the field name aligns with Arrow's default, see e.g. array_into_list_array
+                let field = Arc::new(Field::new("item", inner_type, true));
```

Comment:
this is in theory a breaking change I guess, but I don't know if there's any reason for anyone to be referring to the item field name